Testscripts/Windows/PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1

```diff
@@ -60,12 +60,21 @@ function Main {
         Provision-VMsForLisa -allVMData $allVMData -installPackagesOnRoleNames "none"
         #endregion
 
-        # Add a new line configuration in systemd logind.conf. UserTasksMax Sets the maximum number of OS tasks each user may run concurrently.
-        $setSystemdConfig = "sed -i '`$aUserTasksMax=122880' /etc/systemd/logind.conf"
-        foreach ($vmData in $allVMData) {
-            Run-LinuxCmd -ip $vmData.PublicIP -port $vmData.SSHPort -username "root" `
-                    -password $password -command $setSystemdConfig | Out-Null
+        Write-LogInfo "Getting Systemd Version."
+        $getSystemdVersion = "systemctl --version | head -n1 | awk '{ print `$NF }'"
```

Comment:
some old distro may not use systemctl

Reply:
Thank you. I will double check.

Reply:
@LiliDeng  I tested some distros and paste the results here. Could you suggest some old distros which we are testing?